app/Resources/themes/phirocode/index.html.twig

```diff
@@ -34,22 +34,22 @@
              width="30" height="30" />
       {% endif %}
     </div>
-    <div class="programs"></div>
+    <div class="programs" dir="ltr"></div>
```

Comment:
After adding `dir="ltr` to the html tag it is set for the *whole* page [1]. There should not be a need to set it to indiviual pieces of the website.

[1] https://www.w3.org/International/questions/qa-html-dir

Reply:
That is correct yes. Problem is this should always be ltr wether it is Arabic or English. That is the reason why i hardcoded it in there as well as in some other cases like Code Statistics. It would also be possible to set it dynamically but it was my understanding that this should always be set to ltr.

>Below the html tag, only use the dir attribute on structural elements where the base direction needs to change in order for the text to display correctly.

This is from your [1] so basically we need it in there but wether it is dynamic or not is up to us.

Reply:
As I understand it this would cause program names and other information regarding the program (f.e. time uploaded now, more information after the redesign) being "mirrored" for users of the RTL page.

Is that really expected behaviour?